lib/index.js

```diff
@@ -54,11 +55,14 @@ function VM (opts = {}) {
   if (opts.stateManager) {
     this.stateManager = opts.stateManager
   } else {
-    this.stateManager = new StateManager({
-      trie: opts.state,
-      blockchain: opts.blockchain,
-      common: this._common
-    })
+    var trie = opts.state || new Trie()
+    if (opts.activatePrecompiles) {
+      trie = new Trie()
+      for (var i = 1; i <= 8; i++) {
```

Comment:
Totally irritated, was this one precompile forgotten before and this is an implicit fix (you are iterating up to 8 and not to 7)?

Reply:
looks that way...